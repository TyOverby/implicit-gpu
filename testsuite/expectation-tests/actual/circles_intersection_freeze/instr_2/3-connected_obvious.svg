version https://git-lfs.github.com/spec/v1
oid sha256:1cbaf605a0d6f5fe457082298ba04bb2d6f964184db60ad009d510e04a684744
size 1174
