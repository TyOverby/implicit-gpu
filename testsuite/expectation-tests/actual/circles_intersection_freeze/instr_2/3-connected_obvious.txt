1 line segments

Line Segment 0 
32 points
Clockwise? false
(15.0,20.158604)
(14.0,20.530006)
(13.0,20.794182)
(12.0,20.930517)
(11.069482,20.0)
(11.20582,19.0)
(11.469994,18.0)
(11.841395,17.0)
(12.0,16.673603)
(12.356817,16.0)
(13.0,15.0)
(13.870377,14.0)
(14.0,13.870377)
(15.0,13.0)
(16.0,12.356817)
(16.673603,12.0)
(17.0,11.841395)
(18.0,11.469994)
(19.0,11.20582)
(20.0,11.069482)
(20.930517,12.0)
(20.794182,13.0)
(20.530006,14.0)
(20.158604,15.0)
(20.0,15.326397)
(19.643183,16.0)
(19.0,17.0)
(18.129627,18.0)
(18.0,18.129627)
(17.0,19.0)
(16.0,19.643183)
(15.326397,20.0)
