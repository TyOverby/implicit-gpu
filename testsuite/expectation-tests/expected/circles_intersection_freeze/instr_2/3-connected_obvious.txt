1 line segments

Line Segment 0 
32 points
Clockwise? false
(6.0,11.158606)
(5.0,11.530006)
(4.0,11.79418)
(3.0,11.930518)
(2.0694816,11.0)
(2.2058198,10.0)
(2.4699936,9.0)
(2.841395,8.0)
(3.0,7.6736016)
(3.3568165,7.0)
(4.0,6.0)
(4.8703756,5.0)
(5.0,4.8703756)
(6.0,4.0)
(7.0,3.3568165)
(7.673602,3.0)
(8.0,2.841395)
(9.0,2.4699938)
(10.0,2.2058198)
(11.0,2.0694816)
(11.930518,3.0)
(11.79418,4.0)
(11.530006,5.0)
(11.158606,6.0)
(11.0,6.3263984)
(10.643184,7.0)
(10.0,8.0)
(9.129623,9.0)
(9.0,9.129623)
(8.0,10.0)
(7.0,10.643184)
(6.3263984,11.0)
